suite/testdata/test-suite.yml

```diff
@@ -1,30 +1,51 @@
-iterations: 0
```

Comment:
Both you and @bingobook02 are modding the contents in this file, suggest you create separate files for your own tests and do the mods in there.